crates/biome_js_analyze/tests/specs/nursery/useSingleJsDocAsterisk/valid.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Make sure that valid case files have the special comment at the beginning of the file, so the testing infrastructure will panic if there are diagnostics emitted in those files. I'm from mobile, I don't recall the exact comment